proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
+
+Cons
+
+* Duplicate working group participation agreements need to be signed
+* My have duplicate fees depending on the membership level if both MPWG and CN4J/Jakarta are joined
+* Tends to not commit to longer term backwards compatibility 
+
+== CN4J
+Pros
+
+* Allows for parties interested in both MPWG and CN4J/Jakarta to sign one working group participation agreement
+* Addresses IP gap in existing MP development
+* Tends to focus on stability and backwards compatibility 
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
+
+Cons
+
+* Currently has a higher entry cost and overall overhead to run
+* Currently has no zero entry participation model for larger companies to sponsor specification development
```

Comment:
True, which we discussed at our Committee meetings last week and still needs to be resolved.

Reply:
And what is the currently proposed zero entry participation model for MPWG proposal as it isn't in the proposal.

Reply:
It is in the sponsorship.adoc proposal